wagtail/wagtailsearch/signal_handlers.py

```diff
@@ -1,10 +1,14 @@
 from __future__ import absolute_import, unicode_literals
 
+import logging
+
 from django.db.models.signals import post_delete, post_save
 
-from wagtail.wagtailsearch.backends import get_search_backends
+from wagtail.wagtailsearch.backends import get_search_backends_with_name
 from wagtail.wagtailsearch.index import get_indexed_models
 
+logger = logging.getLogger('wagtail.search.auto_reindex')
```

Comment:
I've named them manually like this a couple of times before (there's one in wagtailcore models and one in the frontendcache backends). I've been doing it partly for forwards-compatibility with the impending drop of the "wagtail" prefix and also to have some control over the submodule name.


I agree that these should be documented somewhere though.